README.md

```diff
@@ -1 +1,40 @@
-# weather-app-js
\ No newline at end of file
+# Weather App
+
+[Click here to view live demo](https://tanzila-abedin.github.io/weather-app-js/)
+
+### This web app was built using JavaScript and allow users to get weather information.
+
+## Built With
+
+- JavaScript
+- Bootstrap
+
+## Getting Started
+
+- Download or clone this repo by using `git clone https://github.com/tanzila-abedin/weather-app-js.git`
+- Run `npm install`
+- Enter the project folder and run locally by using `npx webpack --watch`
```

Comment:
With this command, the user won't be able to start your app locally.

kindly fix the webpack settings and make the user can start the app with the scripts commands in `package .json`

Your scripts command 👇 
![image](https://user-images.githubusercontent.com/31897434/131266180-5b92759e-bf6b-40e9-ae5a-89d5144d108e.png)

The user should be able to run `npm run dev` to start the app locally.